### PR TITLE
Changed scene_system to allow for indirect access using string names of scenes.

### PR DESCRIPTION
**Pull request by Erkle-on-Bones**
Changed SceneManager and Scene to be able to be referenced and accessed through the string of a scene's name, instead of accessing the scene directly. Also added a getName to CharacterInventoryFacade and CharacterManager to allow the possibility of doing this for characters as well.

### CODE REVIEW

File: src/main/java/characters/CharacterManager.java [hunk 1]
```diff
@@ -110,6 +110,10 @@ public void setStatusEffect(String statusName, int duration) {
         }
 
     }
+
+    public String getName(){
```
Comment (wBernie) [inline src/main/java/characters/CharacterManager.java:114]: needs javadoc

File: src/main/java/scene_system/SceneManager.java [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+
+/**
+ * Handles modifications and accessing of information from scene classes, and stores a hashmap
+ * of the scenes in the game, that can be referenced by their names in string form.
+ */
 public class SceneManager {
 
+    private HashMap<String, Scene> sceneList;
 
     public SceneManager(){}
```
Comment (wBernie) [inline src/main/java/scene_system/SceneManager.java:17]: is this still needed?

**Review by wBernie — APPROVED** [review]

File: src/main/java/characters/CharacterManager.java (hunk 1 above)
Comment (Erkle-on-Bones) [reply src/main/java/characters/CharacterManager.java:118]: Added a javadoc

File: src/main/java/scene_system/SceneManager.java (hunk 2 above)
Comment (Erkle-on-Bones) [reply src/main/java/scene_system/SceneManager.java:17]: Yes in order for it to work we need to store all the scenes in scene manager

**Review by edwardsues — APPROVED** [review]
looks good!

File: src/main/java/scene_system/SceneManager.java (hunk 2 above)
Comment (wBernie) [reply src/main/java/scene_system/SceneManager.java:17]: so it didn't show the exact line i commented on, i meant the empty constructor

**Pull request merged**